homeassistant/components/overkiz/config_flow.py

```diff
@@ -97,3 +122,17 @@ async def async_step_dhcp(self, discovery_info: dhcp.DhcpServiceInfo) -> FlowRes
         self._abort_if_unique_id_configured()
 
         return await self.async_step_user()
+
+    async def async_step_reauth(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle reauth."""
+        self._config_entry = self.hass.config_entries.async_get_entry(
+            self.context["entry_id"]
+        )
+
+        if self._config_entry:
```

Comment:
Can the case where `_config_entry` is non here ever happen?

Reply:
It should not be possible

Reply:
You are correct, I did this for mypy satisfaction, however I should have casted it.

Reply:
Since ```self.hass.config_entries.async_get_entry``` returns `ConfigEntry | None` I added this, but I will cast this.